website/docs/_posts/2017-10-05-configuration-options.md

```diff
@@ -530,6 +530,16 @@ Configuration specific for major dependency updates.
 
 Add to this object if you wish to define rules that apply only to major updates.
 
+## matchVersion
```

Comment:
Can you please expand on this documentation a little by showing an example (kind of as seen in issue #1771